backend/db_funcs/cipherFuncs.py

```diff
@@ -1,64 +0,0 @@
-from . import DBConn
```

Comment:
Proč mažem tenhle soubor? Navíc to všechno rozbije. Hned potom, co zkusíš `make`, to zařve obrovskou děsivou chybu.